UnityProject/Assets/Scripts/Core/Chat/EmoteActionManager.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EmoteActionManager : MonoBehaviour
+{
+    [SerializeField]
+    private List<EmoteSO> emotes;
+
+    public static bool FindEmote(string emote, EmoteActionManager instance)
+    {
+		var emoteArray = emote.Split(' ');
+        foreach (var e in instance.emotes)
+        {
+            if(emoteArray[1] == e.emoteName)
+            {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public static void DoEmote(string emote, GameObject player, EmoteActionManager instance)
+    {
+		foreach (var e in instance.emotes)
```

Comment:
indentation looks weird

Reply:
![image](https://user-images.githubusercontent.com/34368774/110512731-2417d900-810e-11eb-8feb-2d87a644e0f2.png)
nothing seems wrong on my end.

Reply:
more than likely, it is combining spaces and tabs. Get an extension to tab the entire thing.